### DISCUSSION TITLE
Branch Created Outside of Schedule

### DISCUSSION THREAD

**Discussion by Kurt-von-Laven** [renovatebot/renovate#18616] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

Kurt-von-Laven/minimal-repro#2 was created at 4:48 p.m. ET (immediately upon approval via the dependency dashboard) despite the schedule dictating that branches be created [before 4 p.m. ET](https://github.com/Kurt-von-Laven/minimal-repro/blob/f9fcf5393bd879ee733b042084bac8cec3e830b3/renovate.json#L8-L9). The repository did not even exist before 4 p.m. ET today since I created it while attempting to create a minimal reproduction for a different bug.

### Relevant debug logs

When I log into Renovate's app dashboard, it doesn't display any repos. I assume this is because I am using Forking Renovate.

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction repository in the bug description

**Comment by viceice** [comment, marked as answer]
dashboard approval overrides any schedule and forces branch creation immediately

**Reply by Kurt-von-Laven** [reply]
Thanks for the quick reply. I was able to find the pertinent guidance in [the `schedule` docs](https://docs.renovatebot.com/configuration-options/#schedule) now that you pointed this out:

> Actions triggered via the [Dependency Dashboard](https://docs.renovatebot.com/configuration-options/#dependencydashboard) are not restricted by a configured schedule.

When I was setting up the Dependency Dashboard, I was reading [the `dependencyDashboard` docs](https://docs.renovatebot.com/configuration-options/#dependencydashboard):

> Just enabling the Dependency Dashboard doesn't change the "control flow" of Renovate. Renovate still creates and manages PRs, and still follows your schedules and rate limits. The Dependency Dashboard gives you extra visibility and control over your updates.

Given this, it would be ideal to make some mention of the fact that, when approval is required, the Dependency Dashboard overrides the schedule in the latter tip and/or [the `dependencyDashboardApproval` docs](https://docs.renovatebot.com/configuration-options/#dependencydashboardapproval). Happy to make a pull request if you agree to hopefully spare you some user confusion in the future.
